pixiedust/display/chart/renderers/google/templates/noapikey.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    <h2>No Google Maps API Key</h2>
+    <p>
+        Some functionality of the Google maps renderer requires an API key.
+    </p>
+    <p>
+        You can obtain an API key from the <a href="https://code.google.com/apis/console/" target="_blank">Google API Console</a>.
+        Once you <a href="https://support.google.com/googleapi/answer/6158862?hl=en&ref_topic=7013279" target="_blank">have an API key</a>,
+        it must be enabled for the <a href="https://console.developers.google.com/projectselector/apis/library" target="_blank">Google Maps Javascript API and the Geocoding API</a>.
+        Then may enter it in the Options dialog.
```

Comment:
you are correct. the message has been rephrased: https://github.com/ibm-cds-labs/pixiedust/commit/d40f4ed9553ef5a4065490f46f4fffbb82e39ce2